README.md

```diff
@@ -1,8 +1,22 @@
 <img src="https://raw.githubusercontent.com/apollographql/space-kit/main/src/illustrations/svgs/rocket1.svg" width="100%" height="144">
 
-[![Tests (all platforms)](https://github.com/apollographql/federation/workflows/Tests%20(all%20platforms)/badge.svg)](https://github.com/apollographql/federation/actions?query=branch%3Amain+workflow%3A%22Tests+%28all+platforms%29%22)
-[![Security audit](https://github.com/apollographql/federation/workflows/Security%20audit/badge.svg)](https://github.com/apollographql/federation/actions?query=workflow%3A%22Security+audit%22)
-[![codecov](https://codecov.io/gh/apollographql/federation/branch/main/graph/badge.svg)](https://codecov.io/gh/apollographql/federation)
+<table>
+    <tbody>
+        <tr>
+            <td></td>
+            <td><img src="https://codecov.io/gh/apollographql/federation/branch/main/graph/badge.svg"> <img src="https://api.netlify.com/api/v1/badges/3a012f93-2d02-41f7-bb2b-848cf005b831/deploy-status"></td>
```

Comment:
These should have `alt=""` tags for accessibility.  The Netlify one has built-in in their dashboard offerings of these badges ([here](https://app.netlify.com/sites/apollo-federation-docs/settings/general#status-badges), I mean), but here's the code they offer there:

```
[![Netlify Status](https://api.netlify.com/api/v1/badges/3a012f93-2d02-41f7-bb2b-848cf005b831/deploy-status)](https://app.netlify.com/sites/apollo-federation-docs/deploys)
```

...which renders as:

[![Netlify Status](https://api.netlify.com/api/v1/badges/3a012f93-2d02-41f7-bb2b-848cf005b831/deploy-status)](https://app.netlify.com/sites/apollo-federation-docs/deploys)

...but will still read "Netlify Status" for a screen-reader.